crates/tx5-connection/src/test.rs

```diff
@@ -37,22 +47,17 @@ impl TestSrv {
 
         for addr in self.server.bind_addrs() {
             if let Ok(r) = Hub::new(
-                br#"{
-                  "iceServers": [
-                    { "urls": ["stun:stun.l.google.com:80"] },
-                    { "urls": ["stun:stun1.l.google.com:80"] }
-                  ]
-                }"#
-                .to_vec(),
+                b"{}".to_vec(),
                 &format!("ws://{addr}"),
                 Arc::new(HubConfig {
                     backend_module: BackendModule::default(),
                     signal_config: Arc::new(tx5_signal::SignalConfig {
                         listener: true,
                         allow_plain_text: true,
-                        max_idle: std::time::Duration::from_secs(1),
+                        max_idle: std::time::Duration::from_secs(max_idle_secs),
```

Comment:
Turns out this was the bit causing my test flakes. The max_idle is now used as the timeout on attempting to connect to webrtc, which was enough for my local system but not CI.